webpack.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
+  entry: './src/index.js',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+};
```

Comment:
### STILL ON THE SETUP ISSUE

- Add this plugin before line 18, just after line 17

```
plugins: [
    new HtmlWebpackPlugin({
      template: './src/index.html',
    }),
  ],
```

Reply:
- [ ] After all the steps, restart your server ` npm start`
- [ ] Your images are broken, please, you'd have to fix this one. 


NOTE: Please, you should set up your webpack correctly. check the webpack tutorial, this will come in handy. :pray: